tools/build/Makefile-Moar.in

```diff
@@ -272,22 +272,22 @@ m-install: m-all tools/build/create-moar-runner.pl tools/build/install-core-dist
 	$(MKPATH) $(DESTDIR)$(PERL6_LANG_DIR)/bin
 	$(MKPATH) $(DESTDIR)$(PERL6_LANG_DIR)/short
 	$(MKPATH) $(DESTDIR)$(PERL6_LANG_DIR)/precomp
-	echo 1 >  $(DESTDIR)$(PERL6_LANG_DIR)/version
+	echo 2 >  $(DESTDIR)$(PERL6_LANG_DIR)/version
 	$(MKPATH) $(DESTDIR)$(PERL6_LANG_DIR)/vendor/dist
 	$(MKPATH) $(DESTDIR)$(PERL6_LANG_DIR)/vendor/sources
 	$(MKPATH) $(DESTDIR)$(PERL6_LANG_DIR)/vendor/resources
 	$(MKPATH) $(DESTDIR)$(PERL6_LANG_DIR)/vendor/bin
 	$(MKPATH) $(DESTDIR)$(PERL6_LANG_DIR)/vendor/short
 	$(MKPATH) $(DESTDIR)$(PERL6_LANG_DIR)/vendor/precomp
-	echo 1 >  $(DESTDIR)$(PERL6_LANG_DIR)/vendor/version
+	echo 2 >  $(DESTDIR)$(PERL6_LANG_DIR)/vendor/version
 	$(MKPATH) $(DESTDIR)$(PERL6_LANG_DIR)/vendor/dist
 	$(MKPATH) $(DESTDIR)$(PERL6_LANG_DIR)/site/dist
 	$(MKPATH) $(DESTDIR)$(PERL6_LANG_DIR)/site/sources
 	$(MKPATH) $(DESTDIR)$(PERL6_LANG_DIR)/site/resources
 	$(MKPATH) $(DESTDIR)$(PERL6_LANG_DIR)/site/bin
 	$(MKPATH) $(DESTDIR)$(PERL6_LANG_DIR)/site/short
 	$(MKPATH) $(DESTDIR)$(PERL6_LANG_DIR)/site/precomp
-	echo 1 >  $(DESTDIR)$(PERL6_LANG_DIR)/site/version
+	echo 2 >  $(DESTDIR)$(PERL6_LANG_DIR)/site/version
```

Comment:
This is probably not a good idea without running the proper upgrade procedure on the repo. We should even remove the "echo 1". We probably only got away with that because all users had already upgraded the repos before I put this into the Makefile. Or it did cause problems and people just wiped their install directory...